review_practice/react-app/src/components/DatePicker/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+import { DayPickerRangeController } from 'react-dates';
+
+import moment from 'moment';
+import 'moment/min/locales';
+
+import * as Style from './style';
+
+export default ({ requestToServer, datePicked, deleteButtonHandle, localeLanguage }) => {
```

Comment:
```suggestion
export default ({ onRequestToServer, onDatePicked, onDeleteButton, localeLanguage }) => {
```

좀 더 props 명칭이 명확해질 필요가 있습니다. 타입스크립트를 사용하지 않기 때문에 동/명사 혼용이 되어 코드에서 무엇을 하는지 알기가 어려워지거든요. 예를 들어 핸들러류의 props는 접두사로 `on-` 을 사용하여 이것은 핸들러라고 명시하는게 좋을 것 같아요. `localeLanguage` 와 같은 명사류는 괜찮습니다 :)